src/api.v2/helpers/pipeline/pipelineConstruct/index.js

```diff
@@ -87,11 +88,23 @@ const withRecomputeDoubletScores = (processingConfig) => {
   return newProcessingConfig;
 };
 
+const getMetadataS3Path = async (experimentId) => {
+  const cellLevelMetadata = await new CellLevelMeta().getMetadataByExperimentIds([experimentId]);
+  if (cellLevelMetadata.length === 0) {
+    return null;
+  }
+  else if (cellLevelMetadata.length === 1) {
+    return cellLevelMetadata[0].id;
+  }
+  else {
+    throw new Error(`Experiment ${experimentId} has multiple cell level metadata files`);
```

Comment:
Nitpicky: What about changing this to a statement of what is the expected situation instead?

Something like `Experiment ${experimentId} cannot have more than one cell level metadata file`.